test/projects/pull-request/pull-request-general-information.html

```diff
@@ -0,0 +1,134 @@
+<!doctype html>
+<html>
+<head>
+  <meta name='viewport' content='width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes'>
+  <title>pull-request-general-information tests</title>
+
+  <script src='../../../src/bower_components/webcomponentsjs/webcomponents-lite.min.js'></script>
+  <script src='../../../src/bower_components/web-component-tester/browser.js'></script>
+  <script src='../../../src/bower_components/iron-test-helpers/mock-interactions.js'></script>
+
+  <!-- Import the element to test -->
+  <link rel='import' href='../../../src/projects/pull-request/pull-request-general-information.html'>
+
+</head>
+<body>
+
+  <test-fixture id='basic'>
+    <template>
+      <pull-request-general-information></pull-request-general-information>
+    </template>
+  </test-fixture>
+
+  <script>
+    suite('<pull-request-general-information>', function() {
+      var examplePull = {
+        body: 'Hello *markdown*',
+        state: 'open',
+        user: { html_url: 'https://example.com/',
+                login: 'txsmith' },
+        head: { label: 'txsmith:pull-branch' },
+        base: { label: 'preview-app:master' },
+        created_at: "2016-05-25T00:31:28Z",
+        assignee: { html_url: 'https://example.com/',
+                    login: 'reviewer' }
+      };
+      var elem;
+
+      setup(function() {
+        elem = fixture('basic');
+        elem.pullRequest = examplePull;
+      });
+
+      test('triggers ajax for comments when PR changes', function(done) {
+        stub('github-authenticated-ajax', {
+          generateRequest: function() {
+            if (this.id == 'ajaxComments') {
+              done();
+            }
+          }
+        });
+
+        elem.pullRequest = {};
+      });
+
+      test('triggers ajax for issues when PR changes', function(done) {
+        stub('github-authenticated-ajax', {
+          generateRequest: function() {
+            if (this.id == 'ajaxIssue') {
+              done();
+            }
+          }
+        });
+
+        elem.pullRequest = {};
+      });
+
+      test('has comments section', function() {
+        assert.isOk(get('pull-request-comments'));
```

Comment:
The tests ensure that the right information is on the page. Apart from this, there was not much to test.